src/native/components/Toast.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { connect } from 'react-redux';
+import { Text, View } from 'react-native';
+import * as Animatable from 'react-native-animatable';
+
+import { delay } from '../../lib/utilities';
+import styles from '../constants/styles';
+
+class Toast extends Component {
+  componentDidUpdate() {
+    if(this.view !== undefined && this.view !== null) {
+      this.animateToast(this.view);
+    }
+  }
+
+  animateToast(viewRef) {
+    viewRef.slideInUp(800)
+      .then(() => delay(3000))
+      .then(() => viewRef.slideOutDown(1000));
```

Comment:
This line is able to create this error:

`Warning: Can only update a mounted or mounting component. This usually means you called setState, replaceState, or forceUpdate on an unmounted component. This is a no-op.`

This is basically replicable by pressing the same wishlist add button really quickly a few times.